libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java

```diff
@@ -667,7 +673,49 @@ public String getTokenFormat() {
 	}
 
 	/**
-	 * Returns the beacon child consumer key .
+	 * Returns the OAuth scopes returned by the token endpoint.
+	 *
+	 * @return scope string.
+	 */
+	public String getScope() {
+		return scope;
+	}
+
+	/**
+	 * Parses the space-delimited scope string into its individual components.
+	 *
+	 * @return Array of scope strings (empty if scope is null/empty).
+	 */
+	public String[] parseScopes() {
```

Comment:
I find Lists to be a lot easier to deal with than Arrays, especially in Kotlin.  Seems like the List `contains` function would be useful for scopes, but you could call `contains()` on the String scope itself as well so really not a big deal.

Edit:  Apparently I should have just kept reading, `hasScope` resolves the above.